test/tests/userobjects/particle_stepper/boris_stepper/tests

```diff
@@ -13,14 +13,38 @@
     [particle]
       type = 'CSVDiff'
       input = 'e_cross_b.i'
-      csvdiff = 'e_cross_b_csv_particle_data_0011.csv'
+      csvdiff = 'e_cross_b_csv_particle_data.csv'
       should_execute = False
       prereq = 'e_cross_b/fields'
       detail = 'and verify particle motion'
     []
   []
   [cyclotron_motion]
     requirement = 'The system shall be capable of accurately capturing the path of charged particles in a perpendicular magnetic field'
+    [convergence]
+      type = PythonUnitTest
+      input = 'test.py'
+      heavy = True
```

Comment:
last time we talked, these tests did not even take that long. If they take 2 seconds or less in total, no need for heavy = true. If they take more than 4, let's leave heavy=true. If we're in the middle, let's chat, but probably leave heavy=true.